pkg/controller/customresource/customresource_test.go

```diff
@@ -4,13 +4,18 @@ import (
 	"fmt"
 	"testing"
 
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
please fix package ordering, it should be in line 8

Reply:
fix comming